src/translation-helpers.ts

```diff
@@ -0,0 +1,58 @@
+export const ErrorCode = {
+    EmptyKeyset: 'EMPTY_KEYSET',
+    EmptyLanguageData: 'EMPTY_LANGUAGE_DATA',
+    KeysetNotFound: 'KEYSET_NOT_FOUND',
+    MissingKey: 'MISSING_KEY',
+    MissingKeyFor0: 'MISSING_KEY_FOR_0',
+    MissingKeyParamsCount: 'MISSING_KEY_PARAMS_COUNT',
+    MissingKeyPlurals: 'MISSING_KEY_PLURALS',
+    NoLanguageData: 'NO_LANGUAGE_DATA',
+} as const;
+
+const codeValues = Object.values(ErrorCode);
+export type ErrorCodeType = (typeof codeValues)[number];
+
+export function mapErrorCodeToMessage(args: {lang?: string; fallbackLang?: string; code?: ErrorCodeType}) {
```

Comment:
if `code === undefined` function may return empty message. Is it correct?

Reply:
it is just for ts satisfying, there is no option to get such result. But, I think u are right, `code` property should be a required